### ISSUE TITLE
Development server not working anymore since 1.9.5

### ISSUE THREAD

**Issue by HybridKat** [parcel-bundler/parcel#1899]
# 🐛 bug report

Since 1.9.5 a simple page with no other modules can't build at all.

## 🎛 Configuration (.babelrc, package.json, cli command)

A simple HTML page `<html><body>Hey!</body></html>,` a completly clean new project.

```js
{
  "name": "Update",
  "version": "1.0.0",
  "description": "",
  "main": "index.html",
  "scripts": {
    "start": "parcel",
    "build": "parcel build"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "dependencies": {
    "parcel-bundler": "^1.9.7"
  }
}
```

## 🤔 Expected Behavior

Page built and http://localhost:1234/ should show "Hey!"

## 😯 Current Behavior

Using Parcel 1.9.5 - 1.9.7 I get a 404 page.  Parcel 1.9.4 works perfectly.

The dist folder is empty, no errors in console.

## 🔦 Context

I wanted to update a project I'm working on.  Since it didn't work after the updates I found out that parcel was not working anymore.  Tried a new project to test if it was a conflict.

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel           | 1.9.7
| Node             | 10.9.0
| npm/Yarn         | npm 6.4.0
| Operating System | Windows 10

**Comment by DeMoorJasper** [comment]
Might be a change in the resolver can you try explicitly using `parcel ./index.html`?

**Comment by HybridKat** [comment]
By using `parcel ./index.html` it works.

Tried changing the "main" in package.json to ./index.html and it quickly flashes the word "Building..." before saying "Built in 35ms" and it's not working.

**Comment by DeMoorJasper** [comment]
Yeah Parcel doesn't throw any errors yet if it has no entrypoints, which is sort of strange. This PR fixes the error throwing: #1848

However we should still figure out why this does not work, as it should

**Comment by sntiagomoreno** [comment]
This is happening in npm version 1.12.3 when using multiple entries.

This throws a 404 when accessing localhost:1234 directly but works when explicitly setting the url to localhost:1234/index.html

```
{
  "name": "parcel-starter",
  "version": "1.0.0",
  "main": "./src/markup/pages/index.hbs",
  "scripts": {
    "dev": "parcel ./src/markup/pages/*.hbs --public-url / --no-cache",
    "build": "parcel build ./src/markup/pages/*.hbs --public-url ./ --no-source-maps"
  },
  "author": "Santiago Moreno",
  "license": "MIT",
  "devDependencies": {
    "parcel-bundler": "^1.12.3",
    "parcel-plugin-clean-dist": "^0.0.6",
    "parcel-plugin-handlebars": "^0.4.4",
    "posthtml-include": "^1.2.0",
    "posthtml-modules": "^0.4.2",
    "sass": "^1.19.0"
  }
}
```
However this works:

```
"scripts": {
    "dev": "parcel ./src/markup/pages/index.hbs --public-url / --no-cache",
    "build": "parcel build ./src/markup/pages/*.hbs --public-url ./ --no-source-maps"
  },
```

Any workaround for this?
